cg/apps/constants.py

```diff
@@ -0,0 +1,7 @@
+BAM_SUFFIX = ".bam"
```

Comment:
don't we have an constants.py already?

Reply:
Yes! I'll move my constants there instead and remove the one I created